### DISCUSSION TITLE
Fail to run the MPFS_VIDEO_KIT_REFERENCE_DESIGN for Libero 2023.2

### DISCUSSION THREAD

**Discussion by wxtang123** [polarfire-soc/polarfire-soc-documentation#412] [Reference designs, FPGA and development kits]
I follow the step below，
-----------------------------
1. Clone or download the repository
2. Open Libero v2023.2
3. Open the execute script dialog (CTRL + U)
4. Execute the "MPFS_VIDEO_KIT_REFERENCE_DESIGN.tcl" script
------------------------------
An error popup 
![image](https://github.com/polarfire-soc/polarfire-soc-documentation/assets/167262881/1e975914-c7d2-4296-a64a-f1950c6278eb)

which is code in MPFS_VIDEO_KIT_H264_REFERENCE_DESIGN.tcl

exec $mss_config_loc -GENERATE -CONFIGURATION_FILE:$src_path/MSS_VIDEO_KIT/H264/MSS_VIDEO_KIT_H264.cfg -OUTPUT_DIR:${src_path}/MSS_VIDEO_KIT/H264/

-------------------------------
setup info as attached file
[libero_setup_info.txt](https://github.com/polarfire-soc/polarfire-soc-documentation/files/15004174/libero_setup_info.txt)

Any hint?

**Comment by hughbreslin** [comment]
Hi @wxtang123 how long is the path to your project? Can you make sure this is as short as possible as it could cause the generation of the MSS to fail

**Reply by wxtang123** [reply]
It 's my first time to try a demo project on Libero.
I've narrow down the problem to fail to open pfsoc_mm.
The fail reason may be lack of a visual studio extension kit.
Thanks for help.
